xds/internal/balancer/cdsbalancer/aggregate_cluster_test.go

```diff
@@ -85,6 +85,10 @@ func (s) TestAggregateClusterSuccess_LeafNode(t *testing.T) {
 					Type:             clusterresolver.DiscoveryMechanismTypeEDS,
 					EDSServiceName:   serviceName,
 					OutlierDetection: json.RawMessage(`{}`),
+					TelemetryLabels: map[string]string{
+						"csm.service_name":           "unknown",
+						"csm.service_namespace_name": "unknown",
+					},
```

Comment:
Can you make a package global and reference it from individual tests.
```
unknownCSMLabels = map[string]string{
	"csm.service_name":           "unknown",
	"csm.service_namespace_name": "unknown",
},
```

I understand this could be a little counter to other comments that we make where we say test logic and state should stay as local as possible, instead of being spread in different places. But these labels, I feel (correct me if I'm wrong), are not very pertinent to the test, and someone reading the test need not know what these labels mean to understand what the test is doing. So, I feel we can make the test tables a tiny bit smaller without losing the readability of them.

This comment applies to other tests as well.

Reply:
Done, check out the line diff of commit :).

Reply:
At first I created symbol in xds/internal/testutils, but that had an import cycle since that imports xdsresource and xdsresource accesses symbol, so I just put in xds/internal.